.github/workflows/deploy.yml

```
@@ -1,55 +1,96 @@
 name: MAN deploy-docker
 
 on:
-#  release:
-#    types:
-#      - published
-  workflow_dispatch:
+ release:
+   types:
+     - published
```

```suggestion
     - published
 workflow_dispatch:
```
Also make manual release possible for testing.